source/algorithms.tex

```diff
@@ -8905,18 +8944,26 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+
 \pnum
-\requires
+\mandates
+All of
+  \begin{itemize}
+  \item \tcode{binary_op(init, *first)},
+  \item \tcode{binary_op(*first, init)},
+  \item \tcode{binary_op(init, init)}, and
+  \item \tcode{binary_op(*first, *first)}
+  \end{itemize}
+  are convertible to \tcode{T}.
```

Comment:
Should this be out-dented to match "All of" above?

Reply:
Yes, thanks.